packages/assets-controllers/CHANGELOG.md

```diff
@@ -7,6 +7,38 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 ## [Unreleased]
 
+## [33.0.0]
+
+### Added
+
+- **BREAKING:** Add `messenger` as a constructor option for `AccountTrackerController` ([#4225](https://github.com/MetaMask/core/pull/4225))
+  - Add `AccountTrackerControllerMessenger` type
+- `NftController` now allows `AccountsController:getAccount`, `AccountsController:getSelectedAccount` as messenger actions and subscribes to the `AccountsController:selectedEvmAccountChange` messenger event ([#4221](https://github.com/MetaMask/core/pull/4221))
```

Comment:
I agree with Elliot's description of the desired behavior, but it doesn't appear that we have type or runtime error guardrails in controller constructor for incompatible messengers. Passing in messengers with incomplete or `never` allowlists into controller constructors seems to work fine.

Runtime errors are thrown, but only when `call` and `subscribe`/`unsubscribe` (or more accurately `#isAllowedAction`, `#isAllowedEvent`) is being invoked.

I'll experiment with this a bit more in the test files since I might just be doing something wrong.